assets/js/script.js

```diff
@@ -200,3 +200,50 @@ allLinks.forEach((link) => {
     $(link.hash).scrollIntoView({ behavior: 'smooth' });
   });
 });
+
+/**
+ * Below are scripts to validate the form before submitting
+ */
+const contactForm = $('#contact-me form');
+const formMessage = $('#form-message');
+const setFormMessage = (message) => {
+  formMessage.innerText = message;
+  if (!formMessage.matches('.form-message')) {
+    formMessage.classList.add('form-message');
+  }
+};
+
+$('#contact-me form button').removeAttribute('disabled');
+contactForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  let error = '';
+  setFormMessage(error);
+  const name = contactForm.elements.name.value;
+  const email = contactForm.elements.email.value;
+  const message = contactForm.elements.message.value;
+  let isValidEmail = false;
+
+  if (!email || !message || !name) {
+    error = 'Error: Ensure that there are no blank fields';
+    return setFormMessage(error);
+  }
+
+  isValidEmail = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?(?:\.[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?)*$/gi.test(
+    email,
+  );
+  if (!isValidEmail) {
+    error = 'Error: Ensure that you have typed the correct email';
+    return setFormMessage(error);
+  }
+
+  if (email !== String(email).toLowerCase()) {
+    error = 'Error: Ensure that the email is typed in lowercase';
+    return setFormMessage(error);
+  }
```

Comment:
- [ ] Great job of adding validation by checking if the email is written in lowercase 👏🏻 However, I suggest you change the color of the text error message to something stronger. You must use a color that can sound more error 🤓 and not white, but make sure the error message matches the chosen template design in Figma (e.g. font, colors, images, tests, margins).

![Screenshot from 2023-02-03 05-45-36](https://user-images.githubusercontent.com/79058364/216510203-552b3967-978c-4a43-bf06-f3e963bb0ca0.png)